xcube/core/geom.py

```diff
@@ -622,31 +621,6 @@ def normalize_geometry(
     raise ValueError(_INVALID_GEOMETRY_MSG)
 
 
-@deprecated(
-    version="0.11.2",
-    reason="convert_geometry() has been" " renamed to normalize_geometry()",
-)
-def convert_geometry(
```

Comment:
Some docstrings of other functions refer to convert_geometry(). See [line 300](https://github.com/xcube-dev/xcube/blob/forman-remove_deprecated_code/xcube/core/geom.py#L300) and [line 447](https://github.com/xcube-dev/xcube/blob/forman-remove_deprecated_code/xcube/core/geom.py#L447). 

Also not sure if docs need to be adjusted [here](https://github.com/xcube-dev/xcube/blob/forman-remove_deprecated_code/docs/source/api.rst?plain=1#L206).

Reply:
Done!

Reply:
Aren't the references to convert_geometry() in the docstrings in [`xcube/core/geom.py` line 300](https://github.com/xcube-dev/xcube/blob/forman-remove_deprecated_code/xcube/core/geom.py#L300) and [`xcube/core/geom.py` line 447](https://github.com/xcube-dev/xcube/blob/forman-remove_deprecated_code/xcube/core/geom.py#L447) confusing? Or am I understanding something wrong here?!